teerex/sgx-verify/src/lib.rs

```diff
@@ -492,6 +492,27 @@ pub fn verify_certificate_chain<'a>(
 	Ok(leaf_cert)
 }
 
+pub fn extract_tcb_info_from_raw_dcap_quote(
+	dcap_quote_raw: &[u8],
+) -> Result<([u8; 6], TcbVersionStatus), &'static str> {
```

Comment:
I think we have defined a type alias for this.

Reply:
You are right, I forgot about that when I was doing a rebase, than you!